bindings/pyroot/pythonizations/CMakeLists.txt

```diff
@@ -95,6 +95,11 @@ foreach(val RANGE ${how_many_pythons})
   add_library(${libname} SHARED ${cpp_sources})
   # Set the suffix to '.so' and the prefix to 'lib'
   set_target_properties(${libname} PROPERTIES  ${ROOT_LIBRARY_PROPERTIES})
+  if(MSVC)
```

Comment:
And the same rational could be applied here:
```
if(APPLE)
elseif(MSVC)
else
```

Reply:
No problem